internal/cmd/asyncapi/command_export.go

```diff
@@ -69,20 +69,23 @@ func newExportCommand(prerunner pcmd.PreRunner) *cobra.Command {
 		Use:   "export",
 		Short: "Create an AsyncAPI specification for a Kafka cluster.",
 	}
+
 	c := &command{AuthenticatedStateFlagCommand: pcmd.NewAuthenticatedStateFlagCommand(cmd, prerunner)}
-	c.RunE = c.export
-	c.Flags().String("file", "asyncapi-spec.yaml", "Output file name.")
-	c.Flags().String("group-id", "consumerApplication", "Consumer Group ID for getting messages.")
-	c.Flags().Bool("consume-examples", false, "Consume messages from topics for populating examples.")
-	c.Flags().String("spec-version", "1.0.0", "Version number of the output file.")
-	c.Flags().String("kafka-api-key", "", "Kafka cluster API key.")
-	c.Flags().String("schema-registry-api-key", "", "API key for Schema Registry.")
-	c.Flags().String("schema-registry-api-secret", "", "API secret for Schema Registry.")
-	c.Flags().String("schema-context", "default", "Use a specific schema context.")
+	cmd.RunE = c.export
+
+	cmd.Flags().String("file", "asyncapi-spec.yaml", "Output file name.")
```

Comment:
are they different, setting flags to c or cmd?

Reply:
`cmd` is "inside of" `c`, so they're the same, but this way will make it easier to detect the "dead" structs I mentioned in the description.